edit/edit.go

```diff
@@ -619,14 +659,25 @@ func edit(ed *Editor, cmd []rune) ([]rune, addr, error) {
 	case 'p':
 		return print(ed, a)
 	case '=':
-		if cmd[1] != '#' {
-			return nil, addr{}, errors.New("unknown command: " + string(c))
-		}
+		var ret string
 		at, err := where(ed, a)
 		if err != nil {
 			return nil, addr{}, err
 		}
-		return []rune(fmt.Sprintf("#%d,#%d", at.from, at.to)), at, nil
+		if len(cmd) == 1 || cmd[1] != '#' {
```

Comment:
Can we move the logic for computing the string version (this if) into a separate function? I want to keep the cases of this switch pretty small, because it's already quite long.